### PR TITLE
Hw1 xs2421

### PR DESCRIPTION
**Pull request by everssun**

### CODE REVIEW

File: xs2421.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Name: Xinghang
+
+UNI: xs2421
+
+GitHub: [link](https://github.com/everssun)
+
+GitHub README: [Link](https://github.com/everssun/everssun/blob/main/README.md)
+
+## Project Proposal
+Financial reinforcement learning tasks often rely on backtest results. However, backtest results often cause some overfitting issues for reasons. One step closer to real-world situation is doing [paper trading](https://www.tradingview.com/support/solutions/43000516466-paper-trading-main-functionality/). My idea of project is to build a library to automate financial stock paper trading.
```
Comment (timkpaine) [inline xs2421.md:12]: move to markdown file as per instructions

File: xs2421.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Project Contributions
+Contribute to [FinRL](https://github.com/AI4Finance-Foundation/FinRL)
```
Comment (timkpaine) [inline xs2421.md:15]: move to `projects/python/FinRL.md`

File: xs2421.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Open Source Development
+
+Name: Xinghang
+
+UNI: xs2421
+
+GitHub: [link](https://github.com/everssun)
+
+GitHub README: [Link](https://github.com/everssun/everssun/blob/main/README.md)
```
Comment (timkpaine) [inline xs2421.md:9]: readme needs an "extra" like a badge or an embeddable app

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: xs2421.md (hunk 3 above)
Comment (everssun) [reply xs2421.md:9]: Hi professor, I think my README page contains a "LinkedIn" badge

**Pull request closed without merging**